Formula/rmate-bash.rb

```diff
@@ -0,0 +1,18 @@
+class RmateBash < Formula
+  desc "Remote TextMate 2 implemented as shell script"
+  homepage "https://github.com/aurora/rmate"
+  url "https://github.com/aurora/rmate.git",
+      :revision => "ace81cf9402ce3b9b96afa1ca9c633d84751be75"
+  version "1.0.2"
+  head "https://github.com/aurora/rmate.git"
+
+  conflicts_with "rmate", :because => "both install `rmate` scripts"
+
+  def install
+    bin.install "rmate"
+  end
+
+  test do
+    system "#{bin}/rmate", "--version"
```

Comment:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (`foo --version` or `foo --help`) are not sufficient, as explained in the [formula cookbook](https://docs.brew.sh/Formula-Cookbook).

In most cases, a good test would involve running a simple test case: run `#{bin}/foo input.txt`.
- Then you can check that the output is as expected (with `assert_equal` or `assert_match` on the output of `shell_output`)
- You can also check that an output file was created, if that is expected: `assert_predicate testpath/"output.txt", :exist?`

Some advice for specific cases:
- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.

Reply:
Hmm, `rmate` can't actually do anything else than opening a GUI program (TextMate), or display help / version information.

It doesn't have any cli-only functionality.

Reply:
The formula for the ruby version of rmate also only does the version check.